frontend/package.json

```diff
@@ -21,6 +21,7 @@
     "prod:playwright": "pnpm i18n:copy-test-locales && pnpm prod",
     "storybook": "nuxt storybook --port=54000",
     "storybook:build": "nuxt storybook build",
+    "storybook:build-for-docs": "NODE_OPTIONS=--openssl-legacy-provider nuxt storybook build",
```

Comment:
This script is run in GitHub CI runner (not Docker), and without `openssl-legacy-provider` it fails.